src/main/resources/templates/db_scripts/alterPrecedingSucceedingTitle.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ${myuniversity}_${mymodule}.preceding_succeeding_title
```

Comment:
I think we need a drop constraint statement right before creation. Because subsequent execution of the script will fail with a 'constraint already exists' error.